### ISSUE TITLE
Reading about recommender systems (theory)

### ISSUE THREAD

**Issue by bbaruque** [jaswellnitz/tourist-chatbot#29]
The section "Recommendation Systems" of the book "Mining of Massive Datasets" is going to be examined in order to deepen the knowledge about recommendation system techniques.

**Comment by bbaruque** [comment]
[bookL (with cgo highlights).pdf](https://github.com/jaswellnitz/tourist-chatbot/files/789787/bookL.with.cgo.highlights.pdf)

**Comment by bbaruque** [comment]
Hi Jasmin,
I found 2 papers that are centered around the general characteristics of recommender in the field of tourism. They're OK to get a general idea of the problem. Although the most interesting read is still the explanation of recommender algorithms that we discussed yesterday.

[[Borras,14] Intelligent tourism recommender systems A survey.pdf](https://github.com/jaswellnitz/tourist-chatbot/files/794207/Borras.14.Intelligent.tourism.recommender.systems.A.survey.pdf)
[[Gavaras,13] Mobile recommender systems in tourism.pdf](https://github.com/jaswellnitz/tourist-chatbot/files/794208/Gavaras.13.Mobile.recommender.systems.in.tourism.pdf)
